build-scripts/SDL_migration.cocci

```diff
@@ -27,10 +27,52 @@
 // declaration
 // @@
 // rule / transformation
-// 
+//
 // So this file is a set of many semantic patches, mostly independant.
 
 
+// SDL_RWread
+@@
+expression e1, e2, e3, e4;
+identifier i;
+@@
+(
+   i = SDL_RWread(e1, e2,
+-  e3, e4);
++  e3 * e4);
```

Comment:
Will `SDL_RWread(rwops, data, 4 + 4, 5 + 5)` be converted correctly?

Reply:
yes, I think so :)

Reply:
this isn't a "preprocessor", it really analyse the code .

Reply:
Yup, you're right!
```patch
-SDL_RWread(rwops, data, 4 + 4, 5 + 5);
+SDL_RWread(rwops, data, (4 + 4) * (5 + 5));
```

Reply:
You might want to special case where one of the two is `1`, as that can just be removed.

Reply:
yes simplification here: 
https://github.com/libsdl-org/SDL/commit/9eaea7d6614c00c865c8059a161631a2668adf12